data/campaigns/Wings_of_Victory/scenarios/01_The_Hunt.cfg

```diff
@@ -0,0 +1,450 @@
+        [/message]
+        [message]
+            speaker=Galun
+            message= _ "... What? ... Sorry, I wasn't listening...."
+        [/message]
+        [message]
+            speaker=Vank
+            message= _ "What is wrong with you? I have not seen you so distracted in ages."
+        [/message]
+        [message]
+            speaker=Galun
+            message= _ "I'm concerned for all of us. The next hatching is near and prey grow scarce."
+        [/message]
+        [message]
+            speaker=Vank
+            message= _ "And if you don't bring your mind back to the hunt, Galun, we'll go hungry right now. We are not Dominants; what can <i>we</i> do about the future?"
+        [/message]
+        [message]
+            speaker=Galun
+            message= _ "I think we must leave Morogor, and seek more land across the Great Ocean."
+        [/message]
+        [message]
+            speaker=Vank
+            message= _ "And just how would you find that land in the Great Ocean without knowing where to go? Drakes have looked, and found nothing but water everywhere."
+        [/message]
+        [message]
+            speaker=Galun
+            message= _ "All the ships passing by must be going somewhere."
+        [/message]
+        [message]
+            speaker=Vank
+            message= _ "That may be, but if you truly aim to lead a flight across the Great Ocean, you'll have to become a Dominant first, otherwise no drake will follow you. (Laughs) And you an <i>Aspirant</i>? Not so long as you dream when the hunt is called..."
+        [/message]
+        [message]
+            speaker=Galun
+            message= _ "Speaking of the hunt, let's take those runners there. You must been distracted or blind yourself, not spotting them before."
+        [/message]
+        [message]
+            speaker=Vank
+            message= _ "... Right ... Hunters! Take wing!"
+        [/message]
+    [/event]
+
+    # Drakes get far enough north to spot the Runner keep
+    [event]
+        name=moveto
+        [filter]
+            side=1
+            x=1-80
+            y=1-32
+        [/filter]
+
+        # TODO may become a tomato surprise? There's no dialog foreshadowing this keep (the start event refers to "runners" but the last two lines imply the "runners" are visible)
+        [terrain]
+            terrain=Ko
+            x,y=50,8
+        [/terrain]
+
+        [terrain]
+            terrain=Co
+            x=49,49,50,50,51,51
+            y= 8, 9, 7, 9, 8, 9
+        [/terrain]
+
+        [unit]
+            side=2
+            type=Outlaw
+            id=Gerth
+            name= _ "Gerth"
+            canrecruit=yes
+            x,y=50,8
+        [/unit]
+
+        {STARTING_VILLAGES 2 10}
+
+        [message]
+            speaker=Gerth
+            message=_ "The damned worms have found us! Send the women and children to the caves. To arms, men."
```

Comment:
"worms" or "wyrms" :P

(Not saying "wyrms" _is_ appropriate, but ... it _might_ be?)

Reply:
Yes, it would be. Thanks.